simulation_parameters/Constants.cs

```diff
@@ -72,19 +72,19 @@ public static class Constants
 
     public const float CELL_BASE_ROTATION = 0.2f;
     public const float CELL_MAX_ROTATION = 0.40f;
-    public const float CELL_MIN_ROTATION = 0.0001f;
-    public const float CELL_MOMENT_OF_INERTIA_DISTANCE_MULTIPLIER = 0.8f;
-    public const float CILIA_ROTATION_FACTOR = 0.035f;
+    public const float CELL_MIN_ROTATION = 0.01f;
+    public const float CELL_MOMENT_OF_INERTIA_DISTANCE_MULTIPLIER = 0.2f;
+    public const float CILIA_ROTATION_FACTOR = 0.020f;
     public const float CILIA_RADIUS_FACTOR_MULTIPLIER = 0.7f;
 
     public const float CELL_COLONY_MAX_ROTATION_MULTIPLIER = 2.5f;
     public const float CELL_COLONY_MIN_ROTATION_MULTIPLIER = 0.05f;
     public const float CELL_COLONY_MAX_ROTATION_HELP = 2.5f;
     public const float CELL_COLONY_MEMBER_ROTATION_FACTOR_MULTIPLIER = 45.0f;
 
-    public const float CILIA_ENERGY_COST = 3.0f;
+    public const float CILIA_ENERGY_COST = 2.0f;
     public const float CILIA_ROTATION_NEEDED_FOR_ATP_COST = 0.03f;
-    public const float CILIA_ROTATION_ENERGY_BASE_MULTIPLIER = 6.0f;
+    public const float CILIA_ROTATION_ENERGY_BASE_MULTIPLIER = 4.0f;
```

Comment:
Does this specific change have an effect? This is really only used to calculate how fast a cell needs to turn before it consumes CILIA_ENERGY_COST. So by lowering this cells need to turn much faster before cilia consumes the full ATP.

Reply:
Yeah, it was very easy to consume ATP by just barely turning prior, which would mean it was easy to drain ATP even if you weren't really planning on turning at all.

Reply:
I suppose that's fine, then. I did test that if you literally move your mouse a few pixels *that* didn't consume much ATP (at least visibly).

Reply:
I didn't actually thoroughly test that specific change though, I tweaked it last night before calling it quits, and forgot to actually test it before uploading today

Reply:
Testing it would definitely be a good thing.